internal/bitwarden/bw/client.go

```diff
@@ -67,23 +68,48 @@ func DisableSync() Options {
 }
 
 func (c *client) CreateObject(obj Object) (*Object, error) {
-	objEncoded, err := c.encode(obj)
-	if err != nil {
-		return nil, err
-	}
+	var out []byte
+	var err error
+	if string(obj.Object) == "attachment" {
+		out, err = c.cmdWithSession("create", string(obj.Object), "--file", obj.File, "--itemid", obj.ItemId).RunCaptureOutput()
+		if err != nil {
+			return nil, err
+		}
+		var tmpObj Object
+		err = json.Unmarshal(out, &tmpObj)
+		if err != nil {
+			return nil, err
+		}
+		result := filterAttachments(tmpObj.Attachments, func(val Attachment) bool {
```

Comment:
It seems we have actually no way to be 100% sure about which attachment is ours correct ? I was hoping the API would put the newly created attachment last, which would help with attachment having the same name.